assets/base.css

```diff
@@ -1097,14 +1097,15 @@ summary::-webkit-details-marker {
 
   .grid--peek .grid__item {
     width: calc(50% - var(--grid-mobile-horizontal-spacing) - 3rem);
+    min-width: 35%;
```

Comment:
Hmm I wonder if should think already about how this might be impacted by Sofia's PR which add a setting to choose the amount of products per row, which is also impacting the amount of cards/slide per page of the slider I think.

Reply:
Worth being aware so we can re-test, but if the mobile settings are only going to allow for 1 or 2 per row this should be fine. If we allow 3, then we'll need to revisit.

Reply:
cc @sofiamatulis so she's aware.

Reply:
Ludo and I are reviewing those PRs too, so it shouldn't get overlooked in 3 column mobile ends up happening there 😄